src/index.js

```diff
@@ -162,7 +132,7 @@ module.exports.createFromBundleConfig = async function createFromBundleConfig(
 
   const configDir = path.dirname(configPath);
 
-  return createBundle(config, configDir, destPath);
```

Comment:
This also totally unbreaks everything, which has been broken since https://github.com/MozillaReality/gltf-bundle/commit/8537b6920bd1643c8795ee4f7694427a303225f8 vOv